assets/js/datasets.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+window.linqs = window.linqs || {};
+window.linqs.utils = window.linqs.utils || {};
+window.linqs.datasets = window.linqs.datasets || {};
+window.linqs.datasets.metadata = window.linqs.datasets.metadata || {};
+window.linqs.datasets.pubs = window.linqs.datasets.pubs || {};
+
+window.linqs.datasets.makeLink = function(path) {
```

Comment:
This seems like it should be in util.
Look over both pubs and datasets for util candidates.